libs/sdk-backend-spi/api/sdk-backend-spi.api.md

```diff
@@ -1482,6 +1482,7 @@ export interface ISettings {
     enableCustomColorPicker?: boolean;
     enableDataSampling?: boolean;
     enableDrilledInsightExport?: boolean;
+    enableDSMForTiger?: boolean;
```

Comment:
I think we should keep FF backend agnostic, so it should be probably just "enableDSM"... And ideally not just "DSM" but the whole words as other FF.